images/aquarium/config.sh

```diff
@@ -105,7 +105,7 @@ if [[ "$kiwi_profiles" == *"Vagrant"* ]]; then
 	chown -R vagrant:vagrant /vagrant
 fi
 
-pip install fastapi uvicorn \
+pip install fastapi==0.63.0 uvicorn==0.13.3 websockets==8.1 \
```

Comment:
should we use `requirements.txt` here?

Reply:
we would have to package `requirements.txt`, and it's nonetheless preferable to rely on the distro provided packages than `pip` as a general rule.